tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27,py33
```

Comment:
ok, err to the future (3.5) if that makes this easier.  what are the issues (summary level)?